freebsd/FreeBSDProcessTable.c

```diff
@@ -248,14 +248,46 @@ void ProcessTable_goThroughEntries(ProcessTable* super) {
 
       proc->priority = kproc->ki_pri.pri_level - PZERO;
 
-      if (String_eq("intr", kproc->ki_comm) && (kproc->ki_flag & P_SYSTEM)) {
-         proc->nice = 0; //@etosan: intr kernel process (not thread) has weird nice value
-      } else if (kproc->ki_pri.pri_class == PRI_TIMESHARE) {
-         proc->nice = kproc->ki_nice - NZERO;
-      } else if (PRI_IS_REALTIME(kproc->ki_pri.pri_class)) {
-         proc->nice = PRIO_MIN - 1 - (PRI_MAX_REALTIME - kproc->ki_pri.pri_level);
-      } else {
-         proc->nice = PRIO_MAX + 1 + kproc->ki_pri.pri_level - PRI_MIN_IDLE;
+      switch (PRI_BASE(kproc->ki_pri.pri_class)) {
+         /* Handling of the below is explained in the FreeBSD base system in:
+          *
```

Comment:
This blank line might be left by accident.

Reply:
Not really, it just looks a bit crammed together without the newline. I will remove it.